File: app/src/main/java/io/euphoria/xkcd/app/impl/connection/ServerSessionView.java
package io.euphoria.xkcd.app.impl.connection;

import io.euphoria.xkcd.app.data.SessionView;

/** Created by Xyzzy on 2017-05-04. */

public interface ServerSessionView extends SessionView {

    /* The (administrative) ID of the server */
    String getServerID();

    /* The (unique) "era" of the server */
    String getServerEra();

}

File: app/src/main/java/io/euphoria/xkcd/app/impl/connection/ServerSessionView.kt
package io.euphoria.xkcd.app.impl.connection

import io.euphoria.xkcd.app.data.SessionView

/** Created by Xyzzy on 2017-05-04.  */
open interface ServerSessionView : SessionView {
    /* The (administrative) ID of the server */
    val serverID: String?

    /* The (unique) "era" of the server */
    val serverEra: String?
}
